cns/restserver/ipam.go

```diff
@@ -300,6 +317,7 @@
 	}
 
 	service.PodIPIDByOrchestratorContext[podInfo.GetOrchestratorContextKey()] = ipconfig.ID
+	logger.Printf("Set PodIPIDByOrchestratorContext item %s with ID %s and IP is %s", podInfo.GetOrchestratorContextKey(), ipconfig.ID, ipconfig.IPAddress)
```

Comment:
Can you prefix it with [setIPConfigAsAllocated]

Reply:
It's not a duplication if we want to ensure line 319 got executed.